server/src/clj_kondo/lsp_server/impl/server.clj

```diff
@@ -114,14 +116,25 @@
                 (.getParentFile))]
     (config-dir dir)))
 
+(defn config-options
+  [arg ^java.io.File config-dir]
+  (let [file (io/file arg)]
+       (if (.isFile file)
```

Comment:
I think this won't work. .isFile only returns true of the file can be resolved against the current working directory.
E.g. 
```
user=> (.isFile (io/file "foobar.edn"))
false
```

A better check may be what we already have in clj-kondo here:

https://github.com/borkdude/clj-kondo/blob/master/src/clj_kondo/impl/core.clj#L66-L67

So if the string doesn't start with `{` or `^`, we assume it must be a file path.